### PR TITLE
[WIP] Iss6 setup

### LINKED ISSUE #6: add setup.py for pip install
Create setup.py and test pip install git+<url>

### PR DESCRIPTION
**Pull request by Naeemkh**
Closes #6

### CODE REVIEW

**Review by mauriciogtec — APPROVED** [review]

File: README.md
Comment (audiracmichelle) [inline README.md:1]: I don't want to get too fixate on this. My understanding is that the name of python package is spacebench, not the name of the project, which remains as SpaCE. @mauriciogtec @Naeemkh

Comment (audiracmichelle) [reply README.md:1]: as such
* the repo name is spacebench
*spacebench stands for SpaCE bench (as in gym bench)
*python package code in /spacebench folder
* SpaCE means Spatial Confounding Environment, and not Spatial Confounding Environment Benchmark

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# SpaCE
-SpaCE, the Spatial Confounding Environment, loads benchmark datasets for causal inference methods tackling spatial confounding.
+# spacebench
+spacebench, the Spatial Confounding Environment Benchmark, loads benchmark datasets for causal inference methods tackling spatial confounding.
```
Comment (audiracmichelle) [inline README.md:2]: would keep original

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+The Spatial Confounding Environment loads benchmark datasets for causal inference that incorporartes spatial confounding. The **SpaCE** datasets contain real confounder and exposure/treatment data inspired by environmental health studies. The synthetic outcome and counterfactuals are generated for causal evaluation. They mimick real outcome data distributions learned with machine learning and neural network methods. Spatial confounding is achieved by masking influential confounders in the learned model. 
 
 ## 🛰️ Code and methods
 
-The code for **SpaCE** data loaders is in the `space` directory. The **SpaCE** datasets are found in the [Harvard Dataverse repository](https://dataverse.harvard.edu/) for transparency and reproducibility.
+The code for **SpaCE** data loaders is in the `spacebench` directory. The **SpaCE** datasets are found in the [Harvard Dataverse repository](https://dataverse.harvard.edu/) for transparency and reproducibility.
```
Comment (audiracmichelle) [inline README.md:12]: yep

File: README.md [hunk 3]
```diff
@@ -20,16 +20,16 @@ To retrieve and generate the data, run the command below in the Terminal:
 ``` sh
 conda env create --file requirements.yaml 
 conda activate space-env
-curl -sSL https://raw.githubusercontent.com/NSAPH-Projects/space/master/data/data.sh | bash -s NN|XGBOOST BINARY|CONT SEED PATH
+curl -sSL https://raw.githubusercontent.com/NSAPH-Projects/space/master/data/data.sh | bash -s [NN or XGBOOST] [BINARY or CONT] SEED PATH
```
Comment (audiracmichelle) [inline README.md:23]: this will only remain in the release that corresponds to CLeaR, correct?

File: README.md [hunk 4]
```diff
@@ -50,7 +50,29 @@ The command downloads the core data and uses it together with the user's input t
 | predictive_model   | xgboost, nn_vcnet |
 | error_model        |                   |
 
+## Installation
```
Comment (audiracmichelle) [inline README.md:53]: Awesome!!

File: README.md [hunk 5]
```diff
@@ -50,7 +50,29 @@ The command downloads the core data and uses it together with the user's input t
...
+- Install the package
+
+``` sh
+pip install -e .
+```
 
 ## 👽 Contact
 
-We welcome contributions and feedback about **SpaCE**. If you have any suggestions or ideas, please open an issue or submit a pull request. Thank you for your interest in our data.
+We welcome contributions and feedback about **spacebench**. If you have any suggestions or ideas, please open an issue or submit a pull request. Thank you for your interest in our data.
```
Comment (audiracmichelle) [inline README.md:78]: cool!

**Review by audiracmichelle — APPROVED** [review]
Approving, although I left some comments and would like to learn your thoughts on it

File: README.md (hunk 1 above)
Comment (mauriciogtec) [reply README.md:2]: So Michelle, you are saying to do it like [`scikit-learn`](https://github.com/scikit-learn/scikit-learn)
where the repo and project are `scikit-learn` but the Python package is `sklearn`.

**Comment by Naeemkh** [comment]
Yes. We can keep SpaCE for all documentation but use spacebench for coding. I will modify it.

File: README.md (hunk 3 above)
Comment (atrisovic) [reply README.md:23]: yes, we'll have a normal api very soon

**Review by atrisovic — COMMENTED** [review]
LGTM
